### PR TITLE
Migrate to AndroidX (and relevant updates)

### PR DESCRIPTION
**Pull request by mkano9**
- AndroidX migration
- Updated targetsdk/compilesdk to 28
- Updated gradle wrapper
- Updated kotlin to 1.3.10

### CODE REVIEW

File: .idea/codeStyleSettings.xml [hunk 1]
```diff
@@ -1,251 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (xiphirx) [inline .idea/codeStyleSettings.xml]: Please don't delete the code style settings.

File: versions.gradle [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
-      targetSdk     : 26,
-      compileSdk    : 26,
+      targetSdk     : 28,
+      compileSdk    : 28,
       minSdk        : 14,
-      buildTools    : '26.0.2',
-      kotlin        : '1.2.41',
-      supportLibrary: '27.0.2',
+      buildTools    : '28.0.3',
+      kotlin        : '1.3.10',
```
Comment (xiphirx) [inline versions.gradle]: Changes to unrelated versions should be split out into a separate pull request.

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.5.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-all.zip
```
Comment (xiphirx) [inline gradle/wrapper/gradle-wrapper.properties]: Changes like this should be split out into a separate pull request.

File: versions.gradle (hunk 2 above)
Comment (miga9) [reply versions.gradle]: Kotlin 1.2.41 was not compiling, so I had to update beyond the current version.

File: .idea/codeStyleSettings.xml (hunk 1 above)
Comment (miga9) [reply .idea/codeStyleSettings.xml]: Ops, not intentional. I will revert this.

File: gradle/wrapper/gradle-wrapper.properties (hunk 3 above)
Comment (miga9) [reply gradle/wrapper/gradle-wrapper.properties]: I don't remember if it was necessary to update this. I will double-check.

File: .idea/codeStyleSettings.xml (hunk 1 above)
Comment (miga9) [reply .idea/codeStyleSettings.xml]: Reverted this. My IDE was automatically removing it.

File: versions.gradle (hunk 2 above)
Comment (Sunshine8705) [reply versions.gradle:8]: Buchanan
